packages/gamut/temp/lint.js

```diff
@@ -0,0 +1,73 @@
+module.exports.default = {
+  create(context) {
+    return {
+      'ImportDeclaration[source.value=/(^@)codecademy(\\u002F)gamut/]': function (
+        node
+      ) {
+        const filename = context.getFilename();
+        const importPath = node.source.value;
+
+        const fileDirectory = filename.split('/packages/')[1].split('/')[0];
+        const importPackage = importPath.split('@codecademy/')[1].split('/')[0];
+
+        if (fileDirectory === importPackage) {
+          context.report({
+            messageId: 'useRelativeImport',
+            node,
+          });
+          return;
+        }
+
+        if (filename.includes('/styleguide/stories')) {
+          if (importPath.includes('/gamut-styles/src')) {
+            context.report({
+              messageId: 'removeSrc',
+              node,
+            });
+          } else if (importPath.includes('/gamut-styles/dist')) {
+            context.report({
+              messageId: 'removeDist',
+              node,
+            });
+          } else if (
+            !importPath.includes('/gamut-styles') &&
+            !importPath.includes('src')
+          ) {
+            context.report({
+              messageId: 'includeSrc',
+              node,
+            });
+          }
+          return;
+        }
+
+        if (node.source.value.includes('src')) {
```

Comment:
On the off chance a file name happens to include `src`?

```suggestion
        if (node.source.value.includes('/src')) {
```